integrations/k8s-using-daemonset/falco-event-generator-deployment.yaml

```
@@ -3,15 +3,19 @@ kind: Deployment
 metadata:
   name: falco-event-generator-deployment
   labels:
-    name: falco-event-generator-deployment
-    app: demo
+    app: falco-event-generator
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+      app: falco-event-generator
   template:
     metadata:
       labels:
         app: falco-event-generator
     spec:
       containers:
       - name: falco-event-generator
-        image: falcosecurity/event-generator:latest
+        image: falcosecurity/event-generator
+        command: ["./event-generator", "run"]
```

```suggestion
        command: ["./event-generator", "run", "syscall", "--loop"]
```
To retain the previous behavior we need 
 - `syscall` (otherwise also the `k8saudit` events collection will run, and probably it will not work because the k8s audit needs to be configured)
 - `--loop`  (as soon as https://github.com/falcosecurity/event-generator/pull/12 gets merged).